### ISSUE TITLE
Error in 'get_evedbtool.sh'

### ISSUE THREAD

**Issue by kimrosebush** [EvEmu-Project/evemu_Crucible#240]
While building from Docker via 'docker-compose -p evemu up --build -d' the following error is produced:

```
Downloading latest EVEDBTool...
Using x86_64 build...
chmod: cannot access 'evedbtool': No such file or directory
ERROR: Service 'server' failed to build: The command '/bin/sh -c cd /src/sql && ./get_evedbtool.sh' returned a non-zero code: 1
```

Reproduction:
1. git clone --branch staging https://github.com/EvEmu-Project/evemu_Crucible
2. docker-compose -p evemu up --build -d
3.  See error

Expected behavior:
Normal startup of the evemu_server container.

System Details:
```
evemu@evemu:~$ uname -a
Linux evemu 5.10.0-12-amd64 #1 SMP Debian 5.10.103-1 (2022-03-07) x86_64 GNU/Linux
evemu@evemu:~$ cat /etc/debian_version
11.2
```
evemu@evemu:~$
 - Docker Build
 - EVEmu 0.8.4-staging-ac55221b

Additional context:
- Executing sql/get_evedbtool.sh produces error:
```
evemu@evemu:~/evemu_Crucible/sql$ ./get_evedbtool.sh
Downloading latest EVEDBTool...
chmod: cannot access 'evedbtool': No such file or directory
```
- Quick fix:
```
cd sql/
wget https://github.com/EvEmu-Project/EVEDBTool/releases/download/0.0.5/evedbtool
chmod +x evedbtool
cd ..
docker-compose -p evemu up --build -d
```

Thanks!

**Comment by jdhirst** [comment]
Fixed by 89d0b792d3b6a726bdf9871e714657c83814372f, please pull and retest 🙂

**Comment by kimrosebush** [comment]
> Fixed by [89d0b79](https://github.com/EvEmu-Project/evemu_Crucible/commit/89d0b792d3b6a726bdf9871e714657c83814372f), please pull and retest 🙂

I get some warnings building eve-core, also docker-compose producing some warnings there too. These were there before this issue, I will open a separate issue for this.

Otherwise built OK and everything is gravy now, thanks @jdhirst !
